Remotion/Web/Test.Shared/TestCspDefaultsProvider.cs

```diff
@@ -0,0 +1,13 @@
+using Remotion.ServiceLocation;
+using Remotion.Web.ContentSecurityPolicy;
+
+namespace Remotion.Web.Test.Shared;
+
+[ImplementationFor(typeof(ICspDefaultsProvider), Lifetime = LifetimeKind.Singleton)]
+public class TestCspDefaultsProvider : CspDefaultsProvider
```

Comment:
We got a pattern for overriding implementations:
1 add a const `Position=0` to the `CspDefualtsProvider` and use this in the attribute
2 add a const `Position=CspDefaultsProvider.Const + 1` and use this in the attribute for TestCspDefaultsProvider.

right now, the order is not defined and i'm suprrised it's not blowing up because of ambiguity